### PR TITLE
[yggtorrent] Use requests module to guess subdomain. Suggested in PR#…

### PR DESCRIPTION
**Pull request by ArthurGarnier**
…4914

Fixes PR #4914

Proposed changes in this pull request:
- Usage of requests to guess the right yggtorrent.is subdomain (which change often)

- [X] PR is based on the DEVELOP branch
- [X] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [X] Read [contribution guide](https://github.com/SickRage/SickRage/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Review by djoole — APPROVED** [review]
I doubt I have the rights to approve a PR, but let's have a try :)

**Comment by ArthurGarnier** [comment]
I guess python-requests have to be added as dependency.

**Comment by ArthurGarnier** [comment]
Already in requirements in fact. Don't know why travis fails on this ?!

File: sickbeard/providers/yggtorrent.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 from __future__ import unicode_literals
 
 import re
+import requests
```
Comment (sharkykh) [inline sickbeard/providers/yggtorrent.py]: @ArthurGarnier 
All you have to do is put the requests import within its import group.
https://travis-ci.org/SickRage/SickRage/builds/421183327#L555

```py
import re

import requests
from requests.compat import urljoin
from requests.utils import dict_from_cookiejar
```

Comment (sharkykh) [reply sickbeard/providers/yggtorrent.py]: @ArthurGarnier 
You still need to remove this line.

**Comment by miigotu** [comment]
I think you just need to enable redirects in the request, rather than every person hitting yggtorrents every time they restart SR even if they dont use the site.

**Comment by miigotu** [comment]
https://github.com/SickRage/SickRage/blob/807c5ae37ecd5af7483f06459c94788ee9db483d/sickbeard/helpers.py#L1463
Redurects are already enabled so Im not sure what happens here. Does the site redirect just back to the home page without post/get data if the wrong url is used?

**Comment by miigotu** [comment]
I have a better idea, ill get back to you in a bit

**Comment by miigotu** [comment]
I have a better idea, ill get back to you in a bit

**Comment by miigotu** [comment]
~~@ArthurGarnier @djoole @sharkykh Can you test my changes in #4952 ? ygg-url-fixes branch~~

Other PR is tested and working. Pushing to master shortly.

**Pull request closed without merging**